index.js

```diff
@@ -235,7 +235,24 @@ module.exports.node = (scriptPath, args, options = {}) => {
 
 	const stdio = normalizeStdio.node(options);
 
-	const {nodePath = process.execPath, nodeOptions = process.execArgv} = options;
+	// Filter out `--inspect` & `--inspect-brk` from default `execArgv`.
+	const defaultExecArgv = process.execArgv.filter(arg => {
```

Comment:
The following might be simpler:

```js
const defaultExecArgv = process.execArgv.filter(arg => arg !== '--inspect' && !arg.startsWith('--inspect=') && arg !== '--inspect-brk' && !arg.startsWith('--inspect-brk='))
```

Reply:
I find this harder to read but happy to update it.